extensions/scarb-snforge-test-collector/src/compilation/test_collector/config.rs

```diff
@@ -55,19 +54,20 @@ impl From<TestExpectation> for ExpectedTestResult {
     }
 }
 
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-pub enum RawForkConfig {
+#[derive(Debug, Clone, PartialEq, Serialize)]
+pub enum ForkConfigString {
     Id(String),
-    Params(RawForkParams),
+    Params(ForkParamsString),
 }
 
-#[derive(Debug, Clone, PartialEq, Serialize, Deserialize)]
-pub struct RawForkParams {
+#[derive(Debug, Clone, PartialEq, Serialize)]
+pub struct ForkParamsString {
     pub url: String,
-    pub block_id: BlockId,
+    pub block_id_type: String,
+    pub block_id_value: String,
```

Comment:
Maybe not the best type to use, but I think it's better than copying structs from `starknet-rs`

Reply:
Perhaps you could add `struct BlockId(String)` to at least increase type safety?

Reply:
I feel like that would be pretending that it is type safe, you can still put anything inside the `BlockId`. I don't think we gain anything by doing it, but maybe I'm missing sth